01-Introduction.Rmd

```diff
@@ -9,8 +9,8 @@ Contributions to base R are possible in a number of different ways. Some of them
 1. Contributing to bug fixing: Refer [Bug Tracking](#BugTrack) and [Reviewing Bugs](#ReviewBugs).
 2. Contributing to translations: Refer [Translations](#).
 3. Testing R before release: Refer [Testing Pre-release R Versions](#TestRVer).
+4. Contributing to documentation and tutorials: Refer [Documenting R](#Doc).
```

Comment:
Is it worth adding a note here that this is a common starting point for new contributors, or add this later?